extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1575,197 +1575,156 @@ describes.realWin(
           );
         });
       });
+    });
 
-      describe('amp-story branching', () => {
```

Comment:
This `describe('amp-story branching', () => {  ...  }` was inside `describe('amp-story pause/resume callbacks', () => { ... }` which I don't think we want, so I just moved it out.

I also moved the toggling of the experiment from the before/afterEach and into the features that are still behind the experiment. The tests for the feature that this PR enables don't need the experiment anymore.